lib/main.js

```diff
@@ -212,6 +212,14 @@ const Hydrogen = {
     autorun(() => {
       this.emitter.emit("did-change-kernel", store.kernel);
     });
+
+    reaction(
+      () => store.runningKernels.size,
+      () => this.emitter.emit("did-start-kernel", store.kernel),
+      // only runs when the new size of runningKernels is increased
+      // $FlowFixMe
+      { equals: (from, to) => from > to }
+    );
```

Comment:
I actually meant to put `from < to`, meaning that it should only react when `store.runningKernels.size` *increases*. So the question is why this worked, the partial answer is it is running any time time the length changes at all.

I'll keep messing around with this to figure it out.